services/recap_service.py

```
@@ -10,18 +11,87 @@
+        if league.groupme_bot_id:
+            try:
+                success = await self.groupme_publisher.send_long_message(
+                    league.groupme_bot_id, 
+                    recap_text
+                )
+                results["groupme"] = success
+                if success:
+                    print(f"Published {recap_type.lower()} to GroupMe for league {league.id}")
+                else:
+                    print(f"Failed to publish {recap_type.lower()} to GroupMe for league {league.id}")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)